hamming/hamming.rb

```diff
@@ -0,0 +1,18 @@
+class Hamming
+  def self.compute(firstStr, secondStr)
+    if firstStr.length != secondStr.length
+      raise ArgumentError, "The lengths of the strands must match."
+    else
+    i, total = 0, 0
+    firstStr = firstStr.split('')
+    secondStr = secondStr.split('')
+    while i<firstStr.length do
```

Comment:
@SurajKoshti `while i<firstStr.length do` should be like with proper format `while i < firstStr.length do`